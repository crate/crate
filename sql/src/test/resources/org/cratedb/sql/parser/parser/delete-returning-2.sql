delete from t1 returning *