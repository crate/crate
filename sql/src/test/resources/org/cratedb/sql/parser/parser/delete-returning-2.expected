org.cratedb.sql.parser.parser.DeleteNode@2d8941f7
statementType: DELETE
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@e49d67c
	isDistinct: false
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@11a06e38

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@3ba42792
		tableName: t1
		updateOrDelete: DELETE
		null
		correlation Name: null
		null
targetTableName: 	
	org.cratedb.sql.parser.parser.TableName@e3d
	t1
returningList: 	
	org.cratedb.sql.parser.parser.ResultColumnList@162200d5

	[0]:	
	org.cratedb.sql.parser.parser.AllResultColumn@1264ab4d
	tableName: null
	exposedName: null
	name: null
	tableName: null
	isDefaultColumn: false
	type: null